src/index.html

```diff
@@ -1,13 +1,651 @@
+                    Hulu
+                  </h3>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="page__section-apply page__section--color apply">
+        <div class="container container--apply">
+          <h2
+            class="page__section-title page__section-title--big apply__title-margin">
+            Vision, Passion, Results
+          </h2>
+
+          <div class="grid grid--desktop">
+            <div class="grid__item grid__item--3-10">
+              <p class="apply__text">
+                We are sure that first-rate job is possible only if all three components are united.
+              </p>
+            </div>
+          </div>
+
+          <div class="apply__button">
+            <a href="" class="button button--apply">
+              Apply
+            </a>
+          </div>
+
+          <div class="apply__big-rectangle"></div>
+
+          <div class="apply__big-rectangle apply__big-rectangle--2"></div>
+
+          <div class="apply__big-rectangle apply__big-rectangle--3"></div>
+
+          <div class="apply__medium-rectangle apply__medium-rectangle--1"></div>
+
+          <div class="apply__medium-rectangle apply__medium-rectangle--2"></div>
+
+          <div class="apply__medium-rectangle apply__medium-rectangle--3"></div>
+
+          <div class="apply__medium-rectangle apply__medium-rectangle--bold apply__medium-rectangle--4"></div>
+
+          <div class="apply__small-rectangle apply__small-rectangle--1"></div>
+
+          <div class="apply__small-rectangle apply__small-rectangle--2"></div>
+
+          <div class="apply__small-rectangle apply__small-rectangle--3"></div>
+
+          <div class="apply__small-rectangle apply__small-rectangle--4"></div>
+
+          <div class="apply__small-rectangle apply__small-rectangle--bold apply__small-rectangle--5"></div>
+        </div>
+      </section>
+    </main>
+
+    <footer class="footer" id="contact-us">
+      <div class="container">
+        <div class="grid grid--tab">
+          <div class="grid__item grid__item--1-5">
+            <div class="footer__info">
+              <h2 class="contact__title">
+                Send us a message
+              </h2>
+
+              <form action="">
+                <input
+                  name="email"
+                  type="email"
+                  placeholder="Your email"
+                  class="footer__field form__field"
+                >
+                <input
+                  name="name"
+                  type="text"
+                  placeholder="Your name"
+                  class="footer__field form__field"
+                >
+                <textarea
+                  name="message"
+                  placeholder="Your message"
+                  rows="1"
+                  class="form__field form__field--textarea"
+                >
+                </textarea>
```

Comment:
```suggestion
               ></textarea>
```
ось так потрібно написати , щоб було видно плейсхолдера